core/src/types/key/common.rs

```diff
@@ -280,6 +280,13 @@ impl super::SigScheme for SigScheme {
         );
     }
 
+    fn from_bytes(_seed: [u8; 32]) -> Self::SecretKey {
+        panic!(
```

Comment:
```suggestion
        unimplemented!(
```
this one will also panic but it conveys more info